src/feature/flow_chart_panel/views/APIKeyModal.tsx

```diff
@@ -1,337 +1,155 @@
 import FamilyHistoryIconSvg from "@src/assets/FamilyHistoryIconSVG";
```

Comment:
Not related just to this file but we should probably rename all of these "APIKey(\*)" related things to "EnvVar(\*)", it's more descriptive since it's just any environment variable and not just API keys